src/DnsRootStack.ts

```diff
@@ -493,6 +493,29 @@ export class DnsRootStack extends cdk.Stack {
         'ns-257.awsdns-32.com',
       ],
     });
+    new Route53.DsRecord(this, 'gn-webform-submissionstorage-dev-ds', {
+      zone: this.cspNijmegenZone,
+      recordName: 'submissionstorage-dev',
+      values: ['33404 13 2 14BD8D96D6C9AE4F5A572F90D9BCE7F74E7A93EFC6F0B66D07EB76B8DC07E8B8'],
+    });
+
+    // Records for webform-submission-storage-accp
+    new Route53.NsRecord(this, 'gn-webform-submission-storage-accp-ns', {
+      zone: this.cspNijmegenZone,
+      recordName: 'submissionstorage-accp',
+      values: [
+        'ns-1253.awsdns-28.org',
+        'ns-238.awsdns-29.com',
+        'ns-741.awsdns-28.net',
+        'ns-1677.awsdns-17.co.uk',
+      ],
+    });
+    new Route53.DsRecord(this, 'gn-webform-submissionstorage-accp-ds', {
+      zone: this.cspNijmegenZone,
+      recordName: 'submissionstorage-accp',
+      values: ['3031 13 2 31A4C2F664BA7C515D835589DC7124EA0034A1187B37BCE69AC8FB63C3956D1E'],
+    });
+
```

Comment:
![afbeelding](https://github.com/GemeenteNijmegen/dns-management/assets/149399985/e836802b-9076-419a-a3a4-2d5f5e3a8e7f)
Is gemakkelijker checken voor mijzelf